apps/dashboard/src/app/(dashboard)/dashboard/connect/ecosystem/create/hooks/use-create-ecosystem.ts

```diff
@@ -23,6 +24,8 @@ export function useCreateEcosystem(
   const { onSuccess, ...queryOptions } = options ?? {};
   const { isLoggedIn } = useLoggedInUser();
   const queryClient = useQueryClient();
+  const token = useAuthHeader();
+  const client = useThirdwebClient(token.data);
```

Comment:
shouldn't we always try to grab the auth token and pass it in? feels like it should happen inside useThirdwebClient no?

Reply:
yes, maybe - however the goal is to no longer need the `useAuthHeader` hook. at all

Reply:
(we need full RSC for that however)